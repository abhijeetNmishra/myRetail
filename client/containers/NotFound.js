import React, { PropTypes } from 'react';

const NotFound = props => (
  <div>
    <h2>Page Not Found !!!</h2>
  </div>
);

export default NotFound;
