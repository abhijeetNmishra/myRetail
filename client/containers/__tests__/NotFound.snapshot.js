/* eslint-disable */
import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from '../NotFound';

test('NotFound renders correctly', () => {
  const notFound = renderer.create(
    <NotFound></NotFound>
  ).toJSON();
  expect(notFound).toMatchSnapshot();
});
