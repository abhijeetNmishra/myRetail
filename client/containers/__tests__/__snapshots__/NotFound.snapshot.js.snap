exports[`test NotFound renders correctly 1`] = `
<div>
  <h2>
    Page Not Found !!!
  </h2>
</div>
`;
